### PR TITLE
Cards reset after some time (Issue #115)

### PR DESCRIPTION
**Pull request by falconis**
animation

### CODE REVIEW

File: .desktop [hunk 1]
```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Encoding=UTF-8
+Name=Link to 
+Type=Link
+URL=file:///home/vishal/Desktop/Collab/OpenCode-Collaborative/index.html
+Icon=text-html
```
Comment (pradeepgangwar) [inline .desktop]: Remove this file

**Comment by falconis** [comment]
Deleted the file.

**Comment by pradeepgangwar** [comment]
Also refer issue you are solving @falconis

**Comment by falconis** [comment]
Ohh yeah...its issue #115. Thanks!

**Comment by aswinzz** [comment]
Provide live link!

**Comment by falconis** [comment]
How to do I do that? :sweat_smile:

**Comment by falconis** [comment]
[link](https://falconis.github.io/OpenCode-Collaborative/)

**Comment by aswinzz** [comment]
LGTM!
@pradeepgangwar can we merge it!

**Pull request merged**